recipes/rdkit/bld.bat

```diff
@@ -0,0 +1,40 @@
+rem surely there's a better way than this
+if "%PY_VER%"=="2.7" (
+	set PYTHON_LIBRARY=python27.lib
+) else if  "%PY_VER%"=="3.4" (
+	set PYTHON_LIBRARY=python34.lib
+) else if  "%PY_VER%"=="3.5" (
+	set PYTHON_LIBRARY=python35.lib
+) else if  "%PY_VER%"=="3.6" (
+	set PYTHON_LIBRARY=python36.lib
+) else (
+	echo "Unexpected version of python"
+	exit 1
+)
+
+cmake ^
+    -G "NMake Makefiles" ^
```

Comment:
`NMake Makefiles` -> `NMake Makefiles JOM`

Reply:
thanks a bunch, these windows builds are really not my thing :)